Source/Core/DolphinQt2/InDevelopmentWarning.cpp

```diff
@@ -59,16 +59,15 @@ InDevelopmentWarning::InDevelopmentWarning(QWidget* parent)
   icon->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
 
   QString body_text = tr(
-      "<p>DolphinQt is a new experimental GUI that is eventually intended to replace "
-      "the current GUI based on wxWidgets. The implementation is <b>very "
-      "incomplete</b>, even basic functionality like changing settings and "
-      "attaching gamepads may be missing or not work at all.</p>\n"
-      "<p>Only developers working on the DolphinQt implementation should use it at "
+      "<p>DolphinQt is a new experimental GUI that is intended to replace "
+      "the current GUI based on wxWidgets. The implementation is <b>currently "
+      "incomplete</b>, so some functionality like changing certain settings may be missing.</p>\n"
```

Comment:
>The implementation is currently incomplete, so some functionality like changing certain settings may be messing.

Suuuuper minor nitpick here - but technically, "like changing certain settings" should be surrounded by commas or in parenthesis, as that is how the sentence is constructed and how one would verbally say it. Since "so some functionality" is short and surrounded by commas, for clarity it would probably be best to use parenthesis, like this:

>The implementation is currently incomplete, so some functionality (like changing certain settings) may be messing.

This is super minor and not important at all, but eh, might as well point it out!